db/views/reporting_months_v02.sql

```diff
@@ -0,0 +1,13 @@
+WITH month_dates AS (
+    SELECT date(generate_series) AS month_date
+    FROM generate_series('2018-01-01', now(), '1 month')
+)
+
+SELECT
+    month_date,
+    EXTRACT(MONTH from month_date) as month,
+    EXTRACT(QUARTER from month_date) as quarter,
+    EXTRACT(YEAR FROM month_date) as year,
+    to_char(month_date, 'YYYY-MM') as month_string,
+    to_char(month_date, 'YYYY-Q') as quarter_string
```

Comment:
Suite.

Reply:
I think it's more valuable for this string to be used for queries and ordering, rather than display. Keeping it `YYYY-Q` makes it much easier to sort - `WHERE quarter_string >= '2020-3'`